spec/models/user_organization_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe UserOrganization, type: :model do
```

Comment:
what will happen if we remove  `type: :model`?

Reply:
The code will still work bhaiya, in case you suggest to remove, I will remove it Bhaiya

Reply:
Sure, why not..

Reply:
removed all the type bhaiya